subsys/net/lib/lwm2m/lwm2m_obj_connmon.c

```diff
@@ -89,10 +91,9 @@ static uint8_t link_quality;
 static uint32_t cellid;
 static uint16_t mnc;
 static uint16_t mcc;
-#if defined(CONFIG_LWM2M_CONNMON_OBJECT_VERSION_1_2)
```

Comment:
Shouldn't we keep those `ifdefs` for variable definitions? We're defining unused variable in case older object version is selected.

Reply:
okay, lets do that :)